### PR TITLE
Fetch remote SCAP resources

### PR DESCRIPTION
**Pull request by Lennonka**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; planned orcharhino 6.4 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.4 or older.

### CODE REVIEW

**Comment by Lennonka** [comment]
@bangelic Can you please perr review this? :)

File: guides/common/modules/ref_remote-scap-resources.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+By default, the SCAP client is configured to ignore the remote resources, which causes the XCCDF rules that rely on the resources to be skipped and result in the `notchecked` status.
+If that is the case, you can see a warning from the SCAP client on your host, such as:
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# grep "foreman_scap_client: WARNING" /var/log/messages | tail -n 2
+Jun  6 09:46:42 _the-user_ foreman_scap_client: WARNING: Datastream component 'scap_org.open-scap_cref_security-data-oval-com.redhat.rhsa-RHEL7.xml.bz2' points out to the remote 'https://access.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2'. Use '--fetch-remote-resources' option to download it.
+Jun  6 09:46:42 _the-user_ foreman_scap_client: WARNING: Skipping 'https://access.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2' file which is referenced from datastream
+----
+
+In connected network environment, you can enable the download of remote resources in {Project}.
```
Comment (bangelic) [inline guides/common/modules/ref_remote-scap-resources.adoc:15]:
```suggestion
In a connected network environment, you can enable the download of remote resources in {Project}.
```

**Review by bangelic — CHANGES_REQUESTED** [review]

**Comment by Lennonka** [comment]
@asteflova Can you please take another look?

File: guides/common/modules/ref_inclusion-of-remote-scap-resources.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+* Name: `fetch_remote_resources`
+* Type: `boolean`
+* Value: `true`
+
++
+For more information, see {ManagingConfigurationsPuppetDocURL}Configuring_Puppet_Smart_Class_Parameters_managing-configurations-puppet[Configuring Puppet Smart Class Parameters] in _{ManagingConfigurationsPuppetDocTitle}_.
+
+ifdef::satellite[]
+If you require remote resources in a disconnected network environment, see ... (xref)
```
Comment (asteflova) [inline guides/common/modules/ref_inclusion-of-remote-scap-resources.adoc:40]: This looks unfinished.

File: guides/common/modules/proc_deploying-a-policy-in-a-host-group-using-ansible.adoc [hunk 3]
```diff
@@ -5,6 +5,9 @@ Use this procedure to deploy a compliance policy in a host group using Ansible.
 
 After you complete the procedure, the Ansible role will install the SCAP client and configure OpenSCAP scans on the host according to the selected compliance policy.
 
+The SCAP content in the compliance policy may require remote resources.
```
Comment (asteflova) [inline guides/common/modules/proc_deploying-a-policy-in-a-host-group-using-ansible.adoc:8]: s/may/might  (per the IBM style guide)

File: guides/common/modules/proc_deploying-a-policy-in-a-host-group-using-puppet.adoc [hunk 4]
```diff
@@ -5,6 +5,9 @@ Use this procedure to configure all the OpenSCAP requirements for a host group.
 
 After you complete the procedure, Puppet will configure OpenSCAP scans on the hosts on the next Puppet run.
 
+The SCAP content in your compliance policy may require remote resources.
```
Comment (asteflova) [inline guides/common/modules/proc_deploying-a-policy-in-a-host-group-using-puppet.adoc:8]: s/may/might  (per the IBM style guide)

File: guides/common/modules/proc_deploying-a-policy-on-a-host-using-ansible.adoc [hunk 5]
```diff
@@ -5,6 +5,9 @@ Use this procedure to configure OpenSCAP scans on a host.
 
 After you complete the procedure, the Ansible role will install the SCAP client and configure OpenSCAP scans on the host according to the selected compliance policy.
 
+The SCAP content in the compliance policy may require remote resources.
```
Comment (asteflova) [inline guides/common/modules/proc_deploying-a-policy-on-a-host-using-ansible.adoc:8]: s/may/might  (per the IBM style guide)

File: guides/common/modules/proc_deploying-a-policy-on-a-host-using-puppet.adoc [hunk 6]
```diff
@@ -5,6 +5,9 @@ Use this procedure to configure all the OpenSCAP requirements for an existing ho
 
 After you complete the procedure, Puppet will configure OpenSCAP scans on the host on the next Puppet run.
 
+The SCAP content in your compliance policy may require remote resources.
```
Comment (asteflova) [inline guides/common/modules/proc_deploying-a-policy-on-a-host-using-puppet.adoc:8]: s/may/might  (per the IBM style guide)

File: guides/common/modules/ref_inclusion-of-remote-scap-resources.adoc (hunk 2 above)
Comment (Lennonka) [reply guides/common/modules/ref_inclusion-of-remote-scap-resources.adoc:40]: I'll remove it here and add it in another PR.

File: guides/common/modules/ref_inclusion-of-remote-scap-resources.adoc [hunk 7]
```diff
@@ -0,0 +1,41 @@
+[id="inclusion-of-remote-scap-resources_{context}"]
+= Inclusion of Remote SCAP Resources
+
+SCAP data streams can reference remote resources, such as OVAL files, that the SCAP client fetches over the internet when it runs on hosts.
+If a data stream requires a remote resource, you can see a warning from the OpenSCAP Scanner tool on your {ProjectServer}, such as:
+[options="nowrap", subs="+quotes,verbatim,attributes"]
```
Comment (maximiliankolb) [inline guides/common/modules/ref_inclusion-of-remote-scap-resources.adoc:6]:
```suggestion

[options="nowrap", subs="+quotes,verbatim,attributes"]
```

**Comment by Lennonka** [comment]
Cherry-picked:
- c2c9fbd8b..737ec36ec  3.7 -> 3.7
- 334aeb8db..71156685b  3.6 -> 3.6

**Pull request merged**